test/test_other.py

```diff
@@ -13130,7 +13130,6 @@ def check_for_es6(filename, expect):
         self.assertContained('foo(arg="hello")', js)
         self.assertContained(['() => 2', '()=>2'], js)
         self.assertContained('const ', js)
-        self.assertContained('let ', js)
```

Comment:
Why did this change?

Reply:
babel doesn't strip comments by default when transpiling.. and we have comments that contains the string ".. let ..." making this hacky check fail.   I think its find to drop it rather than try to get clever here.

Reply:
I see. sgtm